### PR TITLE
Add catalog info for backstage

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  name: test-frame
+  title: Test Frame
+  namespace: quality-community
+  description: |
+    Java library for easy testing of kubernetes deployments and operators using fabric8 api
+  annotations:
+    github.com/project-slug: 'skodjob/test-frame'
+  tags:
+    - java
+    - fabric8
```
Comment (thrix) [inline catalog-info.yaml:14]: @kornys this is not part of the approved tags, can you send an MR to:

https://gitlab.cee.redhat.com/quality-innovators/qe-toolbelt/

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - kubernetes
+    - openshift
+    - maven
```
Comment (thrix) [inline catalog-info.yaml:17]: same for `maven`, `operator-framework` and `junit5`

**Review by thrix — CHANGES_REQUESTED** [review]

**Review by thrix — APPROVED** [review]
LGTM

**Review by im-konge — APPROVED** [review]
Thanks!

**Review by Frawless — APPROVED** [review]

**Pull request merged**